Reference/Cache/index.md

```diff
@@ -10,15 +10,16 @@ In normal environments caching seems to be a pretty standard and easy concept, h
 
 **If you are caching business logic data that changes based on a user's action in the backoffice and you are not using an *ICacheRefresher* then you will need to review your code and update it based on the below documentation.**
 
-## [Retrieving and Adding items in the cache](updating-cache.md)
+## Retrieving and Adding items in the cache
 
-Describes the process of how to get, update and insert items in the cache
+You can [update and insert items in the cache](updating-cache.md).
 
 ## Refreshing/Invalidating cache
 
-### ICacheRefresher
+### [ICacheRefresher](cache-refresher.md)
 
 The standard way to invalidate cache in Umbraco is to implement an `ICacheRefresher`. 
+Ensures that content cache is refreshed among all server nodes participating in a load balanced scenario.
```

Comment:
Not sure I understand what you mean to say here:

Ensuring that the Umbraco content cache is refreshed amongst all server nodes participating in a load balanced scenario.

or

For example: To ensure the content cache is refreshed amongst all server nodes participating in a load balanced scenario.

Reply:
I'll just remove the line.  It is also explain in the sub doc.